readme.md

```diff
@@ -27,8 +27,8 @@ The page should match the design Pixel Perfect: all the sizes, colors and distan
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://Diegocgamarra.github.io/layout_moyo-header/)
+- [TEST REPORT LINK](https://Diegocgamarra.github.io/layout_moyo-header/report/html_report/)
```

Comment:
<img width="626" alt="Captura de Tela 2024-06-17 às 09 30 33" src="https://github.com/mate-academy/layout_moyo-header/assets/22202745/7af8d589-f3eb-4445-8ff3-4445d53974f6">

Reply:
Hello Diego!
Looks like the tests doesn't passed. You can fix this exercises adjusting
- Paddings (left, right): should be 0.
- Font: should be regular not bold.
- The logo position should be slightly to the right.

Please, make sure its fitting just like the report says, use the reference and diffs column to help you (https://diegocgamarra.github.io/layout_moyo-header/report/html_report/)